src/Infrastructure/Event/HelperSetInjectorInterface.php

```diff
@@ -5,13 +5,11 @@
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
 
-namespace SprykerSdk\Sdk\Infrastructure\Command;
+namespace SprykerSdk\Sdk\Infrastructure\Event;
```

Comment:
Please move it to the Injector class. It is not an `Event`. Confusing.

the same for the rest interfaces and classes here. Keep only `Events` in the `Event` directory.